### PR TITLE
Remove standalone screencast setting and related code

### PR DESCRIPTION
**Pull request by antross**
Effectively removes the old screencast in favor of always using the standalone screencast implementation.

### CODE REVIEW

File: src/devtoolsPanel.ts [hunk 1]
```diff
@@ -514,7 +508,7 @@ export class DevToolsPanel {
                 ">
             </head>
             <body>
-                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUri}?experiments=true&theme=${theme}&headless=${enableScreencast}&standaloneScreencast=${standaloneScreencast}&cssMirrorContent=${cssMirrorContent}"></iframe>
+                <iframe id="devtools-frame" frameBorder="0" src="${cdnBaseUri}?experiments=true&theme=${theme}&standaloneScreencast=true&cssMirrorContent=${cssMirrorContent}"></iframe>
```
Comment (hkal) [inline src/devtoolsPanel.ts:511]: We should stage a patch to deprecate the `standaloneScreencast` queryparam in dtfe

**Review by hkal — APPROVED** [review]
LGTM

File: src/devtoolsPanel.ts (hunk 1 above)
Comment (antross) [reply src/devtoolsPanel.ts:511]: @hkal don't we need to keep that param for the DevTools Visual Studio extension (not VS Code)? This extension continues to opt-in so other hosts that don't have their own standalone screencast can use the one built-in with DevTools.

Comment (hkal) [reply src/devtoolsPanel.ts:511]: Oh right - totally slipped my mind. No followup is needed then.

**Pull request merged**